azurelinuxagent/agent.py

```diff
@@ -65,10 +65,11 @@ def __init__(self, verbose, conf_file_path=None):
         if conf.get_logs_console():
             logger.add_logger_appender(logger.AppenderType.CONSOLE, level,
                     path="/dev/console")
-        # See issue #1035
```

Comment:
has the issue described in #1035 been fixed?

Reply:
Fixed Now. Added an addl. test `test_telemetry_logger_verify_not_logging_errors_warnings` to verify if it is fixed.

Reply:
As discussed offline, we need a solid solution for the infinite recursion described in #1035